sormas-app/app/src/androidTest/java/de/symeda/sormas/app/TestHelper.java

```diff
@@ -53,9 +53,10 @@
 
 	public static void initTestEnvironment(boolean setInformantAsActiveUser) {
 		// Initialize a testing context to not operate on the actual database
-		RenamingDelegatingContext context = new RenamingDelegatingContext(InstrumentationRegistry.getTargetContext(), "test_");
+		Context context = InstrumentationRegistry.getInstrumentation().getTargetContext();
```

Comment:
This will always use the existing db on the emulator and afterwards delete it. Would be good to somehow wrap this to use another database location.

Reply:
In the worst case we could mock DatabaseHelper.DATABASE_NAME to provide a different name when testing,

Reply:
DatabaseHelper is used with a prefixed test database now.